client/lib/plans/features-list.tsx

```diff
@@ -1769,6 +1779,22 @@ export const FEATURES_LIST: FeatureList = {
 		getAlternativeTitle: () => '0%',
 		getFeatureGroup: () => FEATURE_GROUP_PAYMENT_TRANSACTION_FEES,
 	},
+	[ FEATURE_50GB_STORAGE_ADD_ON ]: {
+		getSlug: () => PRODUCT_1GB_SPACE,
+		getQuantity: () => 50,
+		getTitle: () => i18n.translate( '50 GB storage add on' ),
```

Comment:
The term we are using is `add-on`. e.g.
<img width="1038" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1842898/eeee6808-75b8-4bc1-9cbf-ebd5d68fe332">

Even though this string seems to not show anywhere, let's still fix it before it gets into the translation pipeline :)

Reply:
[adb4a6b](https://github.com/Automattic/wp-calypso/pull/79041/commits/adb4a6b06f44442917d8c0e7cea7612991659bd1)